packages/lumx-react/src/components/message/Message.tsx

```diff
@@ -0,0 +1,126 @@
+import { mdiAlert, mdiAlertCircle, mdiCheckCircle, mdiInformation } from '@lumx/icons';
+import { ColorPalette, Icon } from '@lumx/react';
+import { COMPONENT_PREFIX, CSS_PREFIX } from '@lumx/react/constants';
+import { IGenericProps, getRootClassName, handleBasicClasses } from '@lumx/react/utils';
+import classNames from 'classnames';
+import React, { ReactElement } from 'react';
+
+/////////////////////////////
+
+enum KindMessage {
+    error = 'error',
+    info = 'info',
+    success = 'success',
+    warning = 'warning',
+}
+
+/**
+ * Defines the props of the component.
+ */
+interface IMessageProps extends IGenericProps {
+    /**
+     * The kind of message.
+     */
+    kind?: string;
+
+    /**
+     * Put a background to the message
+     */
+    hasBackground?: boolean;
+}
```

Comment:
It would be nice to have the `children` props here to document it's use as the message content